flocker/cli/script.py

```diff
@@ -250,9 +250,49 @@ def _changestate_on_nodes(self, deployment, deployment_config,
         return DeferredList(results)
 
 
+@flocker_standard_options
+class CLIOptions(Options):
+    """
+    Command line options for ``flocker`` CLI.
+
+    """
+    longdesc = """flocker does nothing (yet).
+
+    """
+
+    synopsis = ("Usage: flocker [OPTIONS] "
+                "\n"
+                "If you have any issues or feedback, you can talk to us: "
+                "http://docs.clusterhq.com/en/latest/gettinginvolved/"
+                "contributing.html#talk-to-us")
+
+
+@implementer(ICommandLineScript)
+class CLIScript(object):
+    """
+    A command-line script to interact with a cluster via the API.
+    """
+    def main(self, reactor, options):
+        """
+        See :py:meth:`ICommandLineScript.main` for parameter documentation.
+
+        :return: A ``Deferred`` which fires when the deployment is complete or
```

Comment:
This comment is the same as in `DeployScript.main`. Is this `CLIScript` going to be only for deployments? If so, perhaps it should take the place (and name?) of `flocker-deploy`. Else this comment should be changed. It is also at odds with the `longdesc` "flocker does nothing (yet).". I'm not quite sure what the best-practice is for a stub: should the description be honest and say that it does nothing, or say what the plan is? Either way the comments and docstrings should be consistent in this.